bootcfg/cli/group_describe.go

```diff
@@ -11,19 +12,29 @@ import (
 
 // groupDescribeCmd describes a Group.
 var groupDescribeCmd = &cobra.Command{
-	Use:   "describe GROUP_ID",
-	Short: "Describe a machine group",
-	Long:  `Describe a machine group`,
-	Run:   runGroupDescribeCmd,
+	Use:     "describe GROUP_ID",
+	Short:   "Describe a machine group",
+	Long:    `Describe a machine group`,
+	PreRunE: validateGroupDescribeCmd,
```

Comment:
Better to do the validation during the run (like kuebctl and the existing create validation) or use these cobra PreRun things?